invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -119,8 +119,8 @@ def test_drift_electrons_extreme_lifetimes(MChits_and_detsim_params):
 
 def test_diffuse_electrons(MChits_and_detsim_params):
 
-    xs                     = MChits_and_detsim_params.zs
-    ys                     = MChits_and_detsim_params.zs
```

Comment:
This sort of bug is exactly why I prefer the style I suggested earlier:

```python
f = MChits_and_detsim_params
use_fixture(f.xs, f.ys)
```

There are two main problems with the style you have chosen

1. It adds a bunch of pretty pointless lines to the code
2. The signal-to-noise ratio of those lines is veeeery low (which is why you didn't spot the bug immediately).